hera_cal/abscal.py

```diff
@@ -1446,9 +1448,9 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
         return return_obj
 
 
-def cut_bls(datacontainer, bls, bl_cut):
+def cut_bls(datacontainer, bls, min_bl_cut=None, max_bl_cut=None):
```

Comment:
It seems to me like `bls` should be an optional parameter, defaulted to None, an `if bls is None: bls = datacontainer.keys()`

Reply:
bls is not a list of keys, its a dictionary of baseline vectors